src/styles/blocks/_card.scss

```
@@ -0,0 +1,100 @@
+@use "../utils/variables" as *;
+
+.card {
+  width: 198px;
+  display: flex;
+  flex-direction: column;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  background: #fff;
+  box-sizing: content-box;
+
+  &__monitor {
```

Not every product's image will represent a monitor. You need more universal name
```suggestion
  &__image {
```